packages/dapp-svelte-wallet/api/src/lib-wallet.js

```diff
@@ -1,5 +1,7 @@
 // @ts-check
 
+import JSON5 from 'json5';
```

Comment:
What's up with using JSON5?  I'm already a little nervous about proliferation (where by proliferation I mean N>1) of JSON libraries.

Reply:
I'll be moving it to the frontend.  Essentially, JSON5 allows inputting JSON-like objects (the JSON5 standard) without all the nasty quoting requirements.  No observable difference in the parsed results.

Reply:
Discounting the quotation, is there any strong reason to introduce JSON5?

Once there's an established std, small benefits shouldn't justify leaving the std. Large benefits might and often do.

FWIW, I expect we'll be introducing a JSON library anyway, not to deviate from the JSON syntax but to do canonical serialization. So I can't claim N <= 1 purity ;)

Reply:
> Once there's an established std, small benefits shouldn't justify leaving the std. Large benefits might and often do.

> Discounting the quotation, is there any strong reason to introduce JSON5?
> FWIW, I expect we'll be introducing a JSON library anyway, not to deviate from the JSON syntax but to do canonical serialization. So I can't claim N <= 1 purity ;)

I think JSON5 is a valuable ergonomic subset between JSON and Justin.  I'd define it in terms of our PEG stack when we start to do more language stuff.  But again, this is just in the user interfaces (where we do all kinds of things that are not what we'd do in SwingSet).

Reply:
The UI runs under SES, and JSON5 is imported after lockdown.  Only strings are passed to `JSON5.parse`.  I think this makes it a minimal exposure.